airflow/jobs/scheduler_job_runner.py

```diff
@@ -513,34 +567,30 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session) -
                         continue
 
                     task_concurrency_limit: int | None = None
-                    if serialized_dag.has_task(task_instance.task_id):
-                        task_concurrency_limit = serialized_dag.get_task(
-                            task_instance.task_id
-                        ).max_active_tis_per_dag
+                    task_dagrun_concurrency_limit: int | None = None
+                    task_group_concurrency_limit: int | None = None
+                    if serialized_dag.has_task(task_id):
+                        serialized_task = serialized_dag.get_task(task_id)
```

Comment:
Easier to just use `get_task` and catch `TaskNotFound`

Reply:
I just flowed the legacy code.
Maybe if is faster than try-catch, :)